card/src/main/java/com/adyen/checkout/card/data/CardType.kt

```diff
@@ -7,10 +7,44 @@
  */
 package com.adyen.checkout.card.data
 
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
 import java.util.regex.Pattern
 
+@Parcelize
+data class CardType(val txVariant: String, val cardBrand: CardBrand? = null) : Parcelable {
```

Comment:
We shouldn't be able to pass both `txVariant` and `cardBrand` at the same time, it's one or the other